lib/enforce_schema_rules.rb

```diff
@@ -27,12 +27,15 @@ def enforce_column_limits(options = {})
           validates_each(*args) do |record, attr, value|
             limit = record.class.columns_hash[attr.to_s].limit
             if limit
-              message = options[:message] % {:count => limit}
+              message = case
```

Comment:
This is just a style thing (and my personal coding style has definitely changed since writing this), but I'm not a fan of the inline case syntax. I think it's more readable like this:
```ruby
message = case
  when options[:message].is_a?(String) 
    options[:message] % {count: limit}
  when limit == 1 
    options[:message][:one]
  else 
    options[:message][:other] % {:count => limit} 
  end
```